### DISCUSSION TITLE
Is there an extension to display other scratch projects or even better, user-pasted HTML code?

### DISCUSSION THREAD

**Discussion by lndfebfdjo** [TurboWarp/extensions#793] [Q&A] [answered]
The network extension doesn't really work for me, since it just opens the browser when packaged, which is not what I need

**Comment by BlueDome77** [comment, marked as answer]
[IFrame.txt](https://github.com/TurboWarp/extensions/files/12193959/IFrame.txt)
Change to .js after downloading

**Reply by lndfebfdjo** [reply]
Wow, thank you so much! Works perfectly!

**Reply by BlueDome77** [reply]
Please select my comment as the answer to the discussion to properly close it. Thanks!
